### ISSUE TITLE
[VERBOSE-1:callback_cache.cc(132)] Could not parse callback cache, aborting restore

### ISSUE THREAD

**Issue by syfgkjasdkn** [flutter/flutter#22606]
When running
```
flutter run
```
I get
```
Launching lib/main.dart on iPhone 5s in debug mode...
[VERBOSE-1:callback_cache.cc(132)] Could not parse callback cache, aborting restore
```
in https://github.com/syfgkjasdkn/flutter_view_swift

---

`flutter doctor -v`
```plain
[✓] Flutter (Channel beta, v0.8.2, on Mac OS X 10.13.6 17G65, locale en-US)
    • Flutter version 0.8.2 at /Users/asd/Developer/flutter
    • Framework revision 5ab9e70727 (4 weeks ago), 2018-09-07 12:33:05 -0700
    • Engine revision 58a1894a1c
    • Dart version 2.1.0-dev.3.1.flutter-760a9690c2

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at /Users/asd/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 10.0)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.0, Build version 10A255
    • ios-deploy 2.0.0
    • CocoaPods version 1.5.3

[✓] Android Studio (version 3.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 28.0.2
    • Dart plugin version 181.5616
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] VS Code (version 1.27.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 2.18.0

[✓] Connected devices (1 available)
    • iPhone 5s • A989159F-933B-4CF7-90BE-C530C36DE9DE • ios • iOS 12.0 (simulator)

• No issues found!
```

**Comment by syfgkjasdkn** [comment]
Also, after `rm -rf build` and running again, I received
```
Flutter crash report; please file at https://github.com/flutter/flutter/issues.

## command

flutter run

## exception

String: Exit code 1 from: /usr/bin/xcrun simctl install A989159F-933B-4CF7-90BE-C530C36DE9DE /Users/asd/Developer/flutter_examples/flutter_view_swift/build/ios/iphonesimulator/Runner.app:
An error was encountered processing the command (domain=IXUserPresentableErrorDomain, code=1):
This app could not be installed at this time.
Failed to load Info.plist from bundle at path /Users/asd/Library/Developer/CoreSimulator/Devices/A989159F-933B-4CF7-90BE-C530C36DE9DE/data/Library/Caches/com.apple.mobile.installd.staging/temp.yZbKDR/extracted/Payload/Runner.app
Failed to load Info.plist from bundle at path /Users/asd/Library/Developer/CoreSimulator/Devices/A989159F-933B-4CF7-90BE-C530C36DE9DE/data/Library/Caches/com.apple.mobile.installd.staging/temp.yZbKDR/extracted/Payload/Runner.app
Underlying error (domain=MIInstallerErrorDomain, code=35):
	Failed to load Info.plist from bundle at path /Users/asd/Library/Developer/CoreSimulator/Devices/A989159F-933B-4CF7-90BE-C530C36DE9DE/data/Library/Caches/com.apple.mobile.installd.staging/temp.yZbKDR/extracted/Payload/Runner.app

```
#0      runCheckedAsync (package:flutter_tools/src/base/process.dart:251:5)
<asynchronous suspension>
#1      SimControl.install (package:flutter_tools/src/ios/simulators.dart:123:12)
#2      IOSSimulator._buildAndInstallApplicationBundle (package:flutter_tools/src/ios/simulators.dart:384:31)
<asynchronous suspension>
#3      IOSSimulator._setupUpdatedApplicationBundle (package:flutter_tools/src/ios/simulators.dart:353:14)
<asynchronous suspension>
#4      IOSSimulator.startApp (package:flutter_tools/src/ios/simulators.dart:283:15)
<asynchronous suspension>
#5      FlutterDevice.runHot (package:flutter_tools/src/resident_runner.dart:290:54)
#6      _asyncThenWrapperHelper.<anonymous closure> (dart:async/runtime/libasync_patch.dart:77:64)
#7      _rootRunUnary (dart:async/zone.dart:1132:38)
#8      _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#9      _FutureListener.handleValue (dart:async/future_impl.dart:129:18)
#10     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:642:45)
#11     Future._propagateToListeners (dart:async/future_impl.dart:671:32)
#12     Future._complete (dart:async/future_impl.dart:476:7)
#13     _SyncCompleter.complete (dart:async/future_impl.dart:51:12)
#14     _AsyncAwaitCompleter.complete (dart:async/runtime/libasync_patch.dart:28:18)
#15     _completeOnAsyncReturn (dart:async/runtime/libasync_patch.dart:295:13)
#16     getApplicationPackageForPlatform (package:flutter_tools/src/application_package.dart)
#17     _asyncThenWrapperHelper.<anonymous closure> (dart:async/runtime/libasync_patch.dart:77:64)
#18     _rootRunUnary (dart:async/zone.dart:1132:38)
#19     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#20     _FutureListener.handleValue (dart:async/future_impl.dart:129:18)
#21     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:642:45)
#22     Future._propagateToListeners (dart:async/future_impl.dart:671:32)
#23     Future._complete (dart:async/future_impl.dart:476:7)
#24     _SyncCompleter.complete (dart:async/future_impl.dart:51:12)
#25     _AsyncAwaitCompleter.complete (dart:async/runtime/libasync_patch.dart:28:18)
#26     _completeOnAsyncReturn (dart:async/runtime/libasync_patch.dart:295:13)
#27     FlutterProject.fromDirectory (package:flutter_tools/src/project.dart)
#28     _asyncThenWrapperHelper.<anonymous closure> (dart:async/runtime/libasync_patch.dart:77:64)
#29     _rootRunUnary (dart:async/zone.dart:1132:38)
#30     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#31     _FutureListener.handleValue (dart:async/future_impl.dart:129:18)
#32     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:642:45)
#33     Future._propagateToListeners (dart:async/future_impl.dart:671:32)
#34     Future._complete (dart:async/future_impl.dart:476:7)
#35     _SyncCompleter.complete (dart:async/future_impl.dart:51:12)
#36     _AsyncAwaitCompleter.complete (dart:async/runtime/libasync_patch.dart:28:18)
#37     _completeOnAsyncReturn (dart:async/runtime/libasync_patch.dart:295:13)
#38     FlutterProject._readManifest (package:flutter_tools/src/project.dart)
#39     _asyncThenWrapperHelper.<anonymous closure> (dart:async/runtime/libasync_patch.dart:77:64)
#40     _rootRunUnary (dart:async/zone.dart:1132:38)
#41     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#42     _FutureListener.handleValue (dart:async/future_impl.dart:129:18)
#43     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:642:45)
#44     Future._propagateToListeners (dart:async/future_impl.dart:671:32)
#45     Future._complete (dart:async/future_impl.dart:476:7)
#46     _SyncCompleter.complete (dart:async/future_impl.dart:51:12)
#47     _rootRunUnary (dart:async/zone.dart:1132:38)
#48     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#49     _FutureListener.handleValue (dart:async/future_impl.dart:129:18)
#50     Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:642:45)
#51     Future._propagateToListeners (dart:async/future_impl.dart:671:32)
#52     Future._complete (dart:async/future_impl.dart:476:7)
#53     _SyncCompleter.complete (dart:async/future_impl.dart:51:12)
#54     _AsyncAwaitCompleter.complete.<anonymous closure> (dart:async/runtime/libasync_patch.dart:33:20)
#55     _rootRun (dart:async/zone.dart:1124:13)
#56     _CustomZone.run (dart:async/zone.dart:1021:19)
#57     _CustomZone.bindCallback.<anonymous closure> (dart:async/zone.dart:947:23)
#58     _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
#59     _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
#60     _runPendingImmediateCallback (dart:isolate/runtime/libisolate_patch.dart:115:13)
#61     _RawReceivePortImpl._handleMessage (dart:isolate/runtime/libisolate_patch.dart:172:5)
```

## flutter doctor

```
[✓] Flutter (Channel beta, v0.8.2, on Mac OS X 10.13.6 17G65, locale en-US)
    • Flutter version 0.8.2 at /Users/asd/Developer/flutter
    • Framework revision 5ab9e70727 (4 weeks ago), 2018-09-07 12:33:05 -0700
    • Engine revision 58a1894a1c
    • Dart version 2.1.0-dev.3.1.flutter-760a9690c2

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at /Users/asd/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 10.0)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.0, Build version 10A255
    • ios-deploy 2.0.0
    • CocoaPods version 1.5.3

[✓] Android Studio (version 3.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 28.0.2
    • Dart plugin version 181.5616
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] VS Code (version 1.27.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 2.18.0

[✓] Connected devices (1 available)
    • iPhone 5s • A989159F-933B-4CF7-90BE-C530C36DE9DE • ios • iOS 12.0 (simulator)

• No issues found!
```

**Comment by DineshKachhot** [comment]
@syfgkjasdkn did you found any solutions ?

**Comment by syfgkjasdkn** [comment]
@DineshKachhot not yet, no.

**Comment by syfgkjasdkn** [comment]
Although the app in https://github.com/syfgkjasdkn/flutter_view_swift seems to be working fine.

<img width="460" alt="screen shot 2018-10-03 at 1 45 23 pm" src="https://user-images.githubusercontent.com/43219386/46414430-9badd680-c712-11e8-8fa6-40ae51771029.png">

**Comment by DineshKachhot** [comment]
Any update on this issue ?

**Comment by zoechi** [comment]
Duplicate of #22787

**Comment by 1pavanb** [comment]
posted my fix story on #22787
